osu.Server.Spectator/Hubs/IMultiplayerServerMatchCallbacks.cs

```diff
@@ -31,5 +32,32 @@ public interface IMultiplayerServerMatchCallbacks
         /// <param name="room">The room to send the event to.</param>
         /// <param name="user">The user whose state has changed.</param>
         Task UpdateMatchUserState(MultiplayerRoom room, MultiplayerRoomUser user);
+
+        /// <summary>
+        /// Let the hub know that a playlist item has been added.
+        /// </summary>
+        /// <param name="room">The room to send the event to.</param>
+        /// <param name="item">The added item.</param>
+        Task OnPlaylistItemAdded(ServerMultiplayerRoom room, MultiplayerPlaylistItem item);
```

Comment:
All the existing methods specify `MultiplayerRoom` in this interface, would prefer these either do too, or the existing usages are updated to use `ServerMultiplayerRoom`. Probably the latter.

Reasoning is that the purpose of this interface when viewed in isolation becomes a touch uncertain with mixed usage.

Reply:
The latter is the only one possible, because it's actually being used as a `ServerMultiplayerRoom` to access the queue (to validate mods).

Or otherwise I could also re-query the room, but that seems wasteful...

Reply:
Latter is fine 👍